core/src/main/java/org/apache/iceberg/TableScanContext.java

```diff
@@ -54,10 +54,18 @@
     this.toSnapshotId = null;
   }
 
-  private TableScanContext(Long snapshotId, Expression rowFilter, boolean ignoreResiduals,
-                           boolean caseSensitive, boolean colStats, Schema projectedSchema,
-                           Collection<String> selectedColumns, ImmutableMap<String, String> options,
-                           Long fromSnapshotId, Long toSnapshotId) {
+  private TableScanContext(
+      Long snapshotId,
```

Comment:
While I personally like this format, in Iceberg we stick to the version that was here before with all the args grouped together and left justified with the first arg